modules/tinymce/src/themes/silver/test/ts/browser/editor/toolbar/NumberInputTest.ts

```diff
@@ -319,4 +319,22 @@ describe('browser.tinymce.themes.silver.throbber.NumberInputTest', () => {
     TinyUiActions.keystroke(editor, Keys.enter());
     assert.isTrue(editor.hasFocus(), 'after enter editor should have focus');
   });
+
+  it('TINY-9754: changing unit to an invalid unit should not set the size to 0', async () => {
+    const editor = hook.editor();
+    const originalFontSize = '16px';
```

Comment:
I would use a different value and unit because 16 is the default value

Reply:
:+1: , changed